linera-service/src/config.rs

```diff
@@ -89,6 +89,7 @@ impl CommitteeConfig {
                     ValidatorState {
                         network_address: v.network.to_string(),
                         votes: 1,
+                        version_info: Default::default(),
```

Comment:
I'm unsure about this, need to think harder about it tomorrow morning.

Reply:
Did we? :thinking: I remember us being in favour of it.  My only (high-level) concern is that `ValidatorState` generally seems to be used to hold validator _config_ (i.e. immutable information about the validator) which makes the transition to state a bit harder.  Maybe it's not worth it because of that at this late hour.

Reply:
The lowest we can move this right now is `linera-core`.  But since it does also apply to `linera-base`, and we might want to use it from Wasm in the future, I'm inclined to leave it there.